index.bs

```diff
@@ -11029,9 +11029,7 @@ Define the following [=console steps=] with |method|, |args|, and
 
   1. Let |source| be the result of [=get the source=] given [=current Realm Record=].
 
-  1. If |method| is "<code>assert</code>", "<code>error</code>",
-     "<code>trace</code>", or "<code>warn</code>", let |stack| be the [=current
-     stack trace=]. Otherwise let |stack| be null.
+  1. Let |stack| be the [=current stack trace=].
```

Comment:
With this change stack can't be null so we can simplify the next step: 

> the <code>stackTrace</code> field set to |stack| if |stack| is not null, or
> omitted otherwise,

can become

> the <code>stackTrace</code> field set to |stack|

Also I think `stackTrace` is now mandatory in `log.BaseLogEntry`?

Reply:
Ah yes indeed. A bit weird to have an unused / undefined log type here, but that's another topic :) 

Thanks for the patch and sorry about the delay